docs/UserGuide.md

```diff
@@ -260,25 +260,33 @@ Output:
 Here are the list of available commands:
 
 Add: add n\NAME ic\IC_NUMBER dob\DATE_OF_BIRTH ad\ADMISSION_DATE w\WARD [r\REMARK] [t\TAG]...
+   Adds a patient to the address book.
    Example: add n\John Doe ic\T1234567P dob\01/01/2000 ad\25/03/2024 w\A1 r\Has a sweet tooth. t\Diabetes t\FallRisk
 
 Clear: clear
    Clears all entries from the address book.
+   Example: clear
 
 Delete: delete INDEX
+   Deletes the patient at the specified INDEX.
    Example: delete 3
 
 Edit: edit INDEX [n\NAME] [ic\IC_NUMBER] [dob\DATE_OF_BIRTH] [ad\ADMISSION_DATE] [w\WARD] [r\REMARK] [t\TAG]...  
+   Edits the patient at the specified INDEX.
    Example: edit 1 ic\T0123456P t\
 
-Find: find KEYWORD [MORE_KEYWORDS]
-   Example: find John
-
-List: list
-   Lists all patients.
+Find: find n\NAME [MORE_NAMES] OR find ic\IC_NUMBER
```

Comment:
I think the [MORE NAMES] should be omitted:
![image](https://github.com/AY2324S2-CS2103T-F10-1/tp/assets/91833948/271f4258-e720-459a-94e4-3d21ef6c8bc1)

Reply:
Resolved